packages/backend-tasks/src/tasks/LocalTaskWorker.ts

```diff
@@ -40,30 +40,36 @@ export class LocalTaskWorker {
     this.logger.info(
       `Task worker starting: ${this.taskId}, ${JSON.stringify(settings)}`,
     );
-
+    const success = false;
+    let attemptNum = 1;
     (async () => {
-      try {
-        if (settings.initialDelayDuration) {
-          await this.sleep(
-            Duration.fromISO(settings.initialDelayDuration),
-            options?.signal,
+      while (!success) {
+        try {
+          if (settings.initialDelayDuration) {
```

Comment:
I think what freben said makes sense, keeping this outside of the retry loop and having a separate short delay between failed runs. I feel we can pick something somewhat arbitrary for now, like just waiting 1s.